src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -50,11 +51,48 @@
     (normalize-input current v)
     current))
 
+(defn- estimated-fees
+  [{:keys [loading-suggested-routes? fees native-currency-symbol amount receiver]}]
+  [rn/view {:style style/estimated-fees-container}
+   [rn/view {:style style/estimated-fees-content-container}
+    [quo/button
+     {:icon-only?          true
+      :type                :outline
+      :size                32
+      :inner-style         {:opacity 1}
+      :accessibility-label :advanced-button
+      :disabled?           loading-suggested-routes?
+      :on-press            #(js/alert "Not implemented yet")}
+     :i/advanced]]
+   [quo/data-item
+    {:container-style style/fees-data-item
+     :blur?           false
+     :description     :default
+     :icon-right?     false
+     :card?           false
```

Comment:
the default for `card?`, `blur?`, `icon-right?` is `false`

Reply:
Sure, will remove unnecessary props

Reply:
Done!